Bio/Blast/Record.py

```diff
@@ -54,7 +56,15 @@ def __init__(self):
         self.database = ''
         self.database_sequences = None
         self.database_letters = None
-
+        
+        @property
+        def database_letters(self):
+            
+            warnings.warn("Accessing the .database_letters will "
+                          "be deprecated in the next " 
+                          "version of Biopython. ", BiopythonDeprecationWarning)
+            
+            return str(self)
```

Comment:
That return value seems completely wrong - it should be an integer value. The tests ought to catch this...

The standard approach is to have the property proxy a private variable ``self._database_letters``, see e.g. https://github.com/biopython/biopython/blob/biopython-169/Bio/SeqFeature.py#L890